Upstream-Status: Backport

https://crrev.com/c/2216073

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 669790c87d1b6c8616b5337593d8ff43ffe6fdbc Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 26 May 2020 13:20:42 +0000
Subject: [PATCH] ozone: remove x11 headers from accessibility tree formatter
 for Linux

That source file is platform agnostic and used by X11 and
Ozone/X11/Wayland. Remove unneseccary includes.

Test: compile test ozone/wayland only
Change-Id: I9d314988f1dff2ae16043619c947a068229de9f0
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2216073
Auto-Submit: Maksim Sisov <msisov@igalia.com>
Commit-Queue: Martin Robinson <mrobinson@igalia.com>
Reviewed-by: Martin Robinson <mrobinson@igalia.com>
Cr-Commit-Position: refs/heads/master@{#771770}
---
 .../accessibility/accessibility_tree_formatter_auralinux.cc     | 2 --
 1 file changed, 2 deletions(-)

diff --git a/content/browser/accessibility/accessibility_tree_formatter_auralinux.cc b/content/browser/accessibility/accessibility_tree_formatter_auralinux.cc
index 85a308d7591d..b09bf27719fd 100644
--- a/content/browser/accessibility/accessibility_tree_formatter_auralinux.cc
+++ b/content/browser/accessibility/accessibility_tree_formatter_auralinux.cc
@@ -21,8 +21,6 @@
 #include "content/browser/accessibility/accessibility_tree_formatter_utils_auralinux.h"
 #include "content/browser/accessibility/browser_accessibility_auralinux.h"
 #include "ui/accessibility/platform/ax_platform_node_auralinux.h"
-#include "ui/base/x/x11_util.h"
-#include "ui/gfx/x/x11.h"
 
 namespace content {
 
-- 
2.25.1

